flow/Tracing.actor.cpp

```diff
@@ -403,3 +533,350 @@ Span::~Span() {
 		g_tracer->trace(*this);
 	}
 }
+
+OTELSpan& OTELSpan::operator=(OTELSpan&& o) {
```

Comment:
Serializing while moving seems strange to me. Do we need a move constructor at all? @sfc-gh-ljoswiak @sfc-gh-mpilman

Reply:
Also it appears that `OTELSpan& operator=(OTELSpan&& o)` and `OTELSpan(OTELSpan&& o)` have different behavior, which seems strange

Reply:
You mean the behavior is different such that the assignment operator serializes the span whereas the move constructor doesn't? I guess it's a little strange. We use this behavior in loops when we want to make sure the existing span gets serialized before assigning a new one. It takes some work off the implementor because then they don't have to remember to call `span.trace()` or some such function.

I don't know if the move constructor is required. I guess you could remove it and see if anything breaks?